File: src/main/java/de/cloudypanda/main/core/config/CoreConfigManager.java
package de.cloudypanda.main.core.config;

import de.cloudypanda.main.Huntcraft;
import de.cloudypanda.main.common.file.AbstractFileManager;

public class CoreConfigManager extends AbstractFileManager<CoreConfigModel> {
    public CoreConfigManager(String fileName, Huntcraft huntcraft) {
        super(fileName, huntcraft, CoreConfigModel.class);
    }
}

File: src/main/java/de/cloudypanda/main/core/config/CoreConfigManager.kt
package de.cloudypanda.main.core.config;

import de.cloudypanda.main.Huntcraft;
import de.cloudypanda.main.common.config.AbstractFileManager;

class CoreConfigManager(fileName: String, huntcraft: Huntcraft) :
    AbstractFileManager<CoreConfigModel>(fileName, huntcraft, CoreConfigModel::class.java) {
}
